pkg/cli/mirror/list/releases.go

```diff
@@ -122,6 +138,66 @@ func (o *ReleasesOptions) Run(ctx context.Context) error {
 			return err
 		}
 	}
+	return nil
+}
+
+func listChannelsForVersion(ctx context.Context, client cincinnati.Client, o *ReleasesOptions, w io.Writer) error {
+	channels, err := cincinnati.GetChannels(ctx, client, o.Channel)
+	if err != nil {
+		return err
+	}
+
+	if _, err := fmt.Fprintf(w, "Listing channels for version %v.\n\n", o.Version); err != nil {
+		return err
+	}
+	for channel := range channels {
+		if _, err := fmt.Fprintf(w, "%s\n", channel); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func listOCPReleaseVersions() error {
+
+	repo, err := name.NewRepository(OCP_Release_Repo)
+	if err != nil {
+		return err
+	}
+	versionTags, err := remote.List(repo, remote.WithAuthFromKeychain(authn.DefaultKeychain))
+	if err != nil {
+		return err
+	}
+
+	ocpVersions := make(map[string]ReleaseVersion)
+	for _, tag := range versionTags {
+		s := strings.Split(tag, ".")
+		if len(s) > 1 {
+			var r ReleaseVersion
+			r.Major, err = strconv.Atoi(s[0])
+			if err != nil {
+				continue // tag is not $major.$minor.$patch(.*) continue to next tag
+			}
+			r.Minor, _ = strconv.Atoi(s[1])
+			v := fmt.Sprintf("%d.%d", r.Major, r.Minor)
```

Comment:
Maybe just load the ReleaseVersions in a slice from the start and `fmt.Sprintf()` at the end?

Reply:
I'm using the map to consolidate tags like 4.9.13-aarch64, 4.9.25-aarch64, and 4.9.0 into just 4.9. I will add a comment t clarify that.